vulcanai2/tests/models/conftest.py

```diff
@@ -0,0 +1,205 @@
+import pytest
```

Comment:
Can we call this file `test_network_config_fixtures.py` to adhere to the `test_*.py` style of test files?

Reply:
I think to have a shared fixtures, it must be named as `conftest.py` : https://docs.pytest.org/en/latest/writing_plugins.html#conftest-py-plugins

Reply:
ah ok no worries then. thanks for the link!